ui/src/tracks/chrome_slices/index.ts

```diff
@@ -176,16 +175,20 @@ export class ChromeSliceTrack extends Track<Config, Data> {
 
   constructor(args: NewTrackArgs) {
     super(args);
+    this.supportsEnhancing = true;
   }
 
   // Font used to render the slice name on the current track.
   protected getFont() {
-    return '12px Roboto Condensed';
+    sliceHeight = DEFAULT_SLICE_HEIGHT * this.trackState.scaleMultiplier;
+
+    return Math.floor(sliceHeight * (2/3)) + 'px Roboto Condensed';
```

Comment:
nit: is there a specific reason to divide by 3 first instead of this?
```suggestion
    return Math.floor(sliceHeight * 2 / 3) + 'px Roboto Condensed';
```